### PR TITLE
test-report: catch unstashing errors

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -189,14 +189,16 @@ pipeline {
...
-                //         unstash 'testReport'
-                //         powershell script: './CIScripts/GenerateTestReport.ps1'
-                //     } finally {
-                //         stash name: 'testReport', includes: '*.xml,*.html', allowEmpty: true
-                //     }
-                // }
+                script {
+                    try {
+                        unstash 'testReport'
+                        powershell script: './CIScripts/GenerateTestReport.ps1'
```
Comment (krdln) [inline Jenkinsfile]: Can we modify it so errors in `GenerateTestReport` are not treated as no test report? Perhaps
```groovy
def hasReports
try { unstash 'testReport'; hasReports = true }
catch (Exception _) { hasReports = false }
if (hasReports) { ... }
```

**Comment by sodar** [comment]
Tested on pipeline cloned from production pipeline: http://10.84.12.26:8080/job/WinContrail/job/WinCi-ds-test/2

File: Jenkinsfile (hunk 1 above)
Comment (sodar) [reply Jenkinsfile]: `GenerateTestReport` should not fail if there are no test reports. Script checks whether each (currently 1) XML exist.

Comment (krdln) [reply Jenkinsfile]: So I guess it could be moved to `finally` block?

Comment (sodar) [reply Jenkinsfile]: Yes. I will move it there.

**Comment by sodar** [comment]
recheck no bug

**Pull request merged**